TourGuide_Horde/30_ArathiHighlands.lua

```diff
@@ -2,6 +2,7 @@ TourGuide:RegisterGuide("Arathi Highlands (30)", "Stranglethorn Vale (30-31)", "
 return [[
 A Hammerfall |QID|655|
 T Hammerfall |QID|655|
+f Hammerfall
```

Comment:
Nice catch. Lets place this just after accepting Raising Spirits. Does that make better sense?

Reply:
Yes it does, will change.